### PR TITLE
[ONNX] Add converter for FastGelu from Microsoft onnxruntime contrib opset

### PR DESCRIPTION
**Pull request by KJlaccHoeUM9l**
ONNX Runtime supports `FastGelu` in its `contrib opset` ([documentation](https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#com.microsoft.FastGelu) and [code](https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/contrib_ops/cpu/bert/fast_gelu.cc)). Implementation of this operator refers to an approximation for original GELU ([article](https://arxiv.org/pdf/1606.08415.pdf)).
Previously, converter for `Gelu` was added to TVM ONNX frontend, however, there is no such converter for `FastGelu`.
This PR adds this converter to ONNX frontend.

### CODE REVIEW

**Comment by KJlaccHoeUM9l** [comment]
Hello @jwfromm!
Could you review this PR?

File: tests/python/frontend/onnx/test_forward.py [hunk 1]
```diff
@@ -5638,13 +5638,14 @@ def verify_reverse_sequence(x, sequence_lens, batch_axis, time_axis):
     verify_reverse_sequence(x, sequence_lens, 1, 0)
 
 
+@pytest.mark.parametrize("op_name", ["Gelu", "FastGelu"], scope="session")
 @tvm.testing.parametrize_targets
-def test_gelu(target, dev):
+def test_gelu(target, dev, op_name):
```
Comment (echuraev) [inline tests/python/frontend/onnx/test_forward.py:5643]: Let's also extend the tests to check accuracy for `fp16` datatype.

File: python/tvm/relay/frontend/onnx.py [hunk 2]
```diff
@@ -926,10 +926,39 @@ def _impl_v1(cls, inputs, attr, params):
...
+        if inputs[1]:
+            bias = inputs[1]
+            x += bias
+
+        # Declare consts
+        const_dtype = infer_type(x).checked_type.dtype
+        half = _expr.const(0.5, dtype=const_dtype)
+        one = _expr.const(1.0, dtype=const_dtype)
+        const1 = _expr.const(math.sqrt(2 / math.pi), dtype=const_dtype)
+        const2 = _expr.const(0.044715 * math.sqrt(2 / math.pi), dtype=const_dtype)
```
Comment (echuraev) [inline python/tvm/relay/frontend/onnx.py:948]: I took a look at the [ONNX documentation](https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#com.microsoft.FastGelu). And they use hard-coded constants in their implementation. Let's check that for `fp16` we will have the same accuracy with the current implementation as ONNX.

File: tests/python/frontend/onnx/test_forward.py (hunk 1 above)
Comment (KJlaccHoeUM9l) [reply tests/python/frontend/onnx/test_forward.py]: done

File: python/tvm/relay/frontend/onnx.py (hunk 2 above)
Comment (KJlaccHoeUM9l) [reply python/tvm/relay/frontend/onnx.py]: done

File: python/tvm/relay/frontend/onnx.py [hunk 3]
```diff
@@ -926,10 +926,45 @@ def _impl_v1(cls, inputs, attr, params):
...
+            bias_shape = infer_shape(bias)
+            assert len(bias_shape) == 1, "bias term must be a 1D tensor"
+            x += bias
+
+        # Declare consts
+        const_dtype = infer_type(x).checked_type.dtype
+        half = _expr.const(0.5, dtype=const_dtype)
+        one = _expr.const(1.0, dtype=const_dtype)
+        const1 = _expr.const(0.7978845608028654, dtype=const_dtype)  # sqrt(2.0 / PI)
+        const2 = _expr.const(0.0356774081363001, dtype=const_dtype)  # 0.044715 * sqrt(2.0 / PI)
```
Comment (echuraev) [inline python/tvm/relay/frontend/onnx.py:954]: Let's return explicit calculation of these values if it is not affect on the accuracy

**Review by echuraev — APPROVED** [review]
LGTM. Thanks!

**Comment by KJlaccHoeUM9l** [comment]
Hello @AndrewZhaoLuo!
Could you take a look at this PR?
Do you have any additional comments/suggestions?

**Review by AndrewZhaoLuo — APPROVED** [review]

**Pull request merged**